app/src/main/java/com/github/palFinderTeam/palfinder/profile/ProfileUser.kt

```diff
@@ -0,0 +1,47 @@
+package com.github.palFinderTeam.palfinder.profile
+
+import com.github.palFinderTeam.palfinder.utils.PrettyDate
+import java.util.*
+import java.io.Serializable
+
+/**
+ * A class to hold the data for a user to be displayed on the profile activity
+ * Username as unique identifier
+ */
+
+data class ProfileUser(
+    private val username: String,
+    private val name: String,
+    private val surname: String,
+    private val joinDate: Date
+) : Serializable {
+
+    fun getName(): String {
```

Comment:
I think that kotlin has nice syntax shortcut for this kind of things. For instance if you specify the val without private, they are automatically gettable (not a real word sorry).

Reply:
Fixed, thanks